src/roulette.py

```diff
@@ -126,3 +126,124 @@ def get(self, bin: int) -> Bin:
         :rtype: Bin
         """
         return self.bins[bin]
+
+
+class BinBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def buildBins(self, wheel: Wheel) -> None:
+        self.create_straight_bets(wheel)
+
+    @staticmethod
+    def create_straight_bets(wheel: Wheel) -> None:
+        odd_for_straight_bet = 35
```

Comment:
can rename with `straight_bet_odd`? (applies for other bet types)

Reply:
Done. Thanks.